defaults/main.yml

```diff
@@ -1,70 +1,157 @@
+# The name of an appliance container template (pveam)
+pve_lxc_ostemplate_name: debian-10.0-standard_10.0-1_amd64.tar.gz
+    # Optional. It is used only if a name of an appliance container template (pveam) was not defined in the variable pve_lxc_ostemplate_name
+# pve_lxc_ostemplate_url: http://download.proxmox.com/images/system/debian-10.0-standard_10.0-1_amd64.tar.gz
+
+# pve_lxc_ostemplate_src: /mnt/os-images/ubuntu-20.04-standard_20.04-1_amd64.tar.gz   # Optional.
+# pve_lxc_ostemplate_storage: local                                                   # Optional.
+# pve_lxc_ostemplate_content_type: vztmpl                                             # Optional.
+# pve_lxc_ostemplate_timeout: 60 # in seconds                                         # Optional.
+# pve_lxc_ostemplate_force: no                                                        # Optional.
+# pve_lxc_ostemplate_validate_certs: no                                               # Optional.
+# pve_lxc_ostemplate_state: present                                                   # Optional.
+
+
+##############################################
+### Container resources definition section ###
+##############################################
+
+# You can change the timeout for operations of the module according to the performance of your remote host
+# pve_lxc_timeout: 30                             # Optional.
+
+# Validate the node's certificates when creating the container
+# pve_lxc_validate_certs: no                      # Optional.
+
+# By default, we suppose that `inventory_hostname` is the FQDN or the hostname of the host to create, so we set the variable to the hostname.
+# You can arbitrarly define this hostname
+pve_hostname: "{{ inventory_hostname.split('.')[0] }}"
+
+# pve_lxc_vmid: 200                               # Optional.
+
+pve_lxc_description: |                            # Optional.
+  This host is a Debian Buster example container configured via Ansible with:
+  - 1 CPU cores
+  - 512MB of RAM
+  - 16GB of system disk
+  - unprivileged: yes
+  - onboot: no
+
+pve_lxc_root_password: # no default value for security reasons, put yours in a vault.
+pve_lxc_root_authorized_pubkey: "{{ lookup('file', '~/.ssh/id_rsa.pub') }}" # Ansible controller default public key
+
+pve_lxc_cpu_cores: 1                              # Optional.
+# pve_lxc_cpu_limit: 1                            # Optional.
+# pve_lxc_cpu_units: 1000                         # Optional.
+
+pve_lxc_memory: 512                               # Optional.
+# pve_lxc_swap: 512                               # Optional.
+
+# Size (in GB) of the main disk we configure for the LXC
+pve_lxc_disk: 16                                  # Optional.
+pve_lxc_storage: local-lvm
+
+# Start the container when node boot. It is recommended setting in 'yes' when container is in production
+pve_lxc_onboot: no                                # Optional.
+
+# pve_lxc_ip_address: 192.168.33.10               # Optional.
```

Comment:
This seems to be purely due to an error in the module's implementation, since the role simply acts as a gateway from the `pve_lxc_ip_address` variable to the module's `ip_address` parameter.

Furthermore, [it's not clear from the documentation](https://docs.ansible.com/ansible/latest/collections/community/general/proxmox_module.html#parameter-ip_address) what the expected syntax or format of the value is, nor is there any example.

I'll check [the collection repository](https://github.com/ansible-collections/community.general) for any reports on this issue. Otherwise I will report a new one.